ivy/functional/backends/paddle/searching.py

```diff
@@ -119,6 +119,9 @@ def nonzero(
     return res.T
 
 
+@with_supported_dtypes(
+    {"2.6.0 and below": ("in32", "int64", "float32", "float64")}, backend_version
```

Comment:
did you mean for the first to be `int32` not `in32`.

Reply:
It is a typo I am  going to fix it .